packages/database/src/migrations/20210326034445-AddCovidVaccineOverlays-modifies-data.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const ROOT_MAP_OVERLAY_ID = '5f88d3a361f76a2d3f000004';
+
+const COUNTRIES = ['WS', 'FJ', 'NR'];
+
+const MAP_OVERLAY_GROUP_BY_COUNTRY = {
+  WS: {
+    id: '5f88d3a261f76a2d3f000001', // COVID-19 Samoa
+  },
+  FJ: {
+    id: generateId(),
+    name: 'COVID-19 Fiji',
+    code: 'COVID19_Fiji',
+  },
+  NR: {
+    id: generateId(),
+    name: 'COVID-19 Nauru',
+    code: 'COVID19_Nauru',
+  },
+}
+
+const MAP_OVERLAY_TEMPLATES = {
+  Total_Number_Of_People_1st_Dose_Covid_Vaccine: {
+    name: 'Total number of people received 1st dose of COVID-19 vaccine',
+    userGroup: 'COVID-19',
+    dataElementCode: 'COVIDVac4',
+    measureBuilderConfig: {"entityAggregation": {"aggregationType": "REPLACE_ORG_UNIT_WITH_ORG_GROUP", "dataSourceEntityType": "village", "aggregationEntityType": "village"}},
```

Comment:
```suggestion
    measureBuilderConfig: {"entityAggregation": {"aggregationType": "REPLACE_ORG_UNIT_WITH_ORG_GROUP", "dataSourceEntityType": "district", "aggregationEntityType": "village"}},
```

Here we need to change it for Nauru and Fiji. See [comment from Slack](https://beyondessential.slack.com/archives/C01BXE4KP5K/p1616969807002900)

Reply:
Thanks, changed to district level, see screenshot